pkg/monitoring/hyperconverged/collectors/controllers.go

```diff
@@ -0,0 +1,18 @@
+package collectors
+
+import (
+	"github.com/rhobs/operator-observability-toolkit/pkg/operatormetrics"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func SetupCollectors(cli client.Client, namespace string) error {
```

Comment:
can you move this login under pkg/monitoring/hyperconverged/metrics/metrics.go? then instead of calling  SetupCollectors in cmd/hyperconverged-cluster-operator/main.go you can call only SetupMetrics